app/controllers/unattended_controller.rb

```diff
@@ -18,13 +18,29 @@ class UnattendedController < ApplicationController
   # all of our requests should be returned in text/plain
   after_action :set_content_type
 
-  # this actions is called by each operatingsystem post/finish script - it notify us that the OS installation is done.
   def built
     logger.info "#{controller_name}: #{@host.name} is Built!"
+    logger.debug "Log lines from the OS installer:"
+    logger.debug(extract_build_log)
```

Comment:
Does a single call to `logger.debug` make more sense here so that the logs are displayed as one log entry?

Do we want to save these logs (success case) also in Foreman's database so we can display them in the UI?

Reply:
Good call, also put that into block to prevent evaluation in non-debug level.

This is intentional, I actually use that column to indicate there was no error (when it is `NULL`). If we want to save also successful logs, then I'd need to introduce second column. Since I plan to add support for remote syslog logging via proxy in the future to get all Anaconda/OS installer logs, I think it's not necessary.